cli/src/commands/generate/components/node.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
I think we should definitely remove this. I know there will be errors to then fix, but it will stop the plague of `any` getting out of control. I've left some other comments with suggestions of alternatives, but I'm happy to discuss further if you disagree

Reply:
Revisiting later